docs/mount.md

```diff
@@ -59,7 +59,8 @@ mounts:
 - location: "~"
   9p:
     # Supported security models are "passthrough", "mapped-xattr", "mapped-file" and "none".
-    # 🟢 Builtin default: "mapped-xattr"
+    # "mapped-xattr" and "mapped-file" are useful for persistent chown but incompatible with symlinks.
+    # 🟢 Builtin default: "none" (since Lima v0.12.1)
```

Comment:
(Maybe this should be rather in v0.13.0)

Reply:
Yes, I feel that changing defaults merits at least a minor version bump.

Reply:
Changed to v0.13.0.

v0.12.1 will be cancelled.